### ISSUE TITLE
the performance of peddy in a vcf calling from exome sequencing

### ISSUE THREAD

**Issue by N-damo** [brentp/peddy#62]
the question may sound stupid but as the title, is  it  possible to use*** exome sequencing*** result to do ***paternity testing*** using peddy?

**Comment by brentp** [comment]
yes, peddy is built with exome sequencing in mind, so the sites are almost all in the exome.
